app/models/spree/payment/processing.rb

```diff
@@ -240,6 +236,9 @@ def handle_response(response, success_state, failure_state)
             if response.cvv_result
               self.cvv_response_code = response.cvv_result['code']
               self.cvv_response_message = response.cvv_result['message']
+              if self.cvv_response_message.present?
+                return self.require_authorization!
+              end
```

Comment:
This is the main place that this PR actually changes logic. This is the first one. We now move the payment to the `requires_authorization` state instead of the `pending` state (even though `pend` is passed as the `success_state`. The response comes back as a success, but the state that the payment should end up in depends on whether the response indicates that authorization is required. 

The only case where we're using the `cvv_response_message` field (as far as I can tell) is to store the authorization URL from Stripe.